xml/storage_iscsi.xml

```diff
@@ -1324,10 +1324,10 @@ o- / ........................................................... [...]
   o- xen-pvscsi ......................................... [Targets: 0]
 /></screen>   
    <para>
-    The output indicates that there is now a file-based back-store, under the
+    The output indicates that there is now a file-based backstore, under the
```

Comment:
Merriam-Webster doesn't know the word, it's not defined in our terminology, and I've personally never heard it before. I think it might be a rather unusual compound and would stick to either keeping the `-` or ` ` for this one. (Same for the multiple mentions below.